src/applications/mhv-medications/containers/Prescriptions.jsx

```diff
@@ -472,10 +476,21 @@ const Prescriptions = () => {
             data-testid="Title-Notes"
           >
             When you share your medications list with providers, make sure you
-            also tell them about your allergies and reactions to medications. If
-            you print or download this list, we’ll include a list of your
-            allergies.
+            also tell them about your allergies and reactions to medications.{' '}
+            {!showAllergiesContent && (
+              <span>
+                If you print or download this list, we’ll include a list of your
+                allergies.
+              </span>
+            )}
           </p>
+          {showAllergiesContent && (
+            <div className="vads-u-margin-bottom--3">
+              <a href="/my-health/medical-records/allergies" rel="noreferrer">
```

Comment:
Would it be possible to use just an <a> tag and apply the styles directly? Just for semantics.

Reply:
let me try 👍

Reply:
tried to add the `className="vads-u-margin-bottom--3"` to the `<a>` tag directly but it didn't apply the style.